roles/openshift_node/tasks/node_system_container.yml

```diff
@@ -12,6 +12,13 @@
     path: "/etc/origin/openvswitch"
     mode: '0750'
 
+# TODO: remove when system container is fixed to not include it
```

Comment:
should we change the system container to create this directory or is it fine to ensure it exists from the installer?  If the system container needs write access to /var/lib/kubelet then it should exist somewhere, so I think it is fine to leave this part anyway.